fold-and-r.csl

```diff
@@ -119,8 +119,8 @@
         </else-if>
         <else-if type="article-journal">
           <group delimiter=" ">
-            <text macro="title" prefix="'" suffix="'"/>
-            <text term="in" suffix=" "/>
+            <text macro="title" quotes="true"/>
+            <text term="in" prefix=", " suffix=" "/>
```

Comment:
```<text macro="title" quotes="true" suffix=","/>
<text term="in"/>
<text macro="journal"/>
```

the way you have this set up right now, you're getting a space between quotation mark and comma because of the group delimiter.